rpcs3/util/vm_native.cpp

```diff
@@ -464,7 +464,7 @@ namespace utils
 		, m_size(utils::align(size, 0x10000))
 	{
 #ifdef _WIN32
-		m_handle = ensure(::CreateFileMappingW(INVALID_HANDLE_VALUE, nullptr, PAGE_EXECUTE_READWRITE, 0, m_size, nullptr));
+		m_handle = ensure(::CreateFileMappingW(INVALID_HANDLE_VALUE, nullptr, PAGE_EXECUTE_READWRITE, 0, ::narrow<DWORD>(m_size), nullptr));
```

Comment:
That is actually a bug, you see it sets 0 on the highbits argument.

Reply:
?

Reply:
These win32 functions split u64 arguments into two arguments: one for lower 32 bits and one for upper 32 bits, here the upper 32bits arg is always 0. This is where the bits should go.

Reply:
I guess i solved it somehow.